src/cleanup_rules/swift/rules.toml

```diff
@@ -769,6 +769,39 @@ not_contains = [
     )""",
 ]
 
+[[rules.filters]]
+not_enclosing_node = "(value_argument_label) @va"
+
+[[rules]]
```

Comment:
I think we won't need this rule at all right? 

Since now anything under `value_argument_label` does not get transformed (as per the update to the above rule)

Reply:
Yes, but we need this to clean for arguments right?